src/Magnum/Math/Frustum.h

```diff
@@ -198,21 +198,51 @@ template<class T> class Frustum {
 
         /** @brief Left plane */
         constexpr Vector4<T> left() const { return _data[0]; }
+        /**
+         * @overload
+         * @m_since_latest
+         */
+        Vector4<T> left() { return _data[0]; }
 
         /** @brief Right plane */
         constexpr Vector4<T> right() const { return _data[1]; }
+        /**
+         * @overload
+         * @m_since_latest
+         */
+        Vector4<T> right() { return _data[1]; }
 
         /** @brief Bottom plane */
         constexpr Vector4<T> bottom() const { return _data[2]; }
+        /**
+         * @overload
+         * @m_since_latest
+         */
+        Vector4<T> bottom() { return _data[2]; }
 
         /** @brief Top plane */
         constexpr Vector4<T> top() const { return _data[3]; }
+        /**
+         * @overload
+         * @m_since_latest
+         */
+        Vector4<T> top() { return _data[3]; }
 
         /** @brief Near plane */
         constexpr Vector4<T> near() const { return _data[4]; }
+        /**
+         * @overload
+         * @m_since_latest
+         */
+        Vector4<T> near() { return _data[4]; }
 
         /** @brief Far plane */
         constexpr Vector4<T> far() const { return _data[5]; }
+        /**
+         * @overload
+         * @m_since_latest
+         */
+        Vector4<T> far() { return _data[5]; }
```

Comment:
It's usually that the order is non-const first, const marked as overload, so for consistency

```cpp
        /** 
         * @brief Far plane 
         * @m_since_latest
         */
        Vector4<T> far() { return _data[5]; }
        constexpr Vector4<T> far() const { return _data[5]; } /**< @overload */
```

Same for others. (Yes, it'll be slightly weird that the main function is documented as new and the overload not, but I don't think that's a serious problem.)